### PR TITLE
fix: TT-336 Fix format Duration column

### PR DESCRIPTION
**Pull request by edgardavid2015**
# Problem
The Duration column format inside the Reports section is not correct. This format is presenting the data in hours and minutes.

wrong_duration_format

# Solution
This pull request retrieves the previous format of the Duration column. This format displays the data in hours or multiples of hours.

correct_duration_format

### CODE REVIEW

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 1]
```diff
@@ -28,11 +29,29 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
       },
       {
         extend: 'excel',
+        exportOptions: {
+          format: {
+            body: (data, row, column, node) => {
+              return column === 3 ?
```
Comment (faustocv) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:35]: We call it magic number. It's a code smell. Please give it a meaningful variable name

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 2]
```diff
@@ -28,11 +29,29 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
...
+          }
+        },
         text: 'Excel',
         filename: `time-entries-${formatDate(new Date(), 'MM_dd_yyyy-HH_mm', 'en')}`
       },
       {
         extend: 'csv',
+        exportOptions: {
+          format: {
+            body: (data, row, column, node) => {
```
Comment (faustocv) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:48]: This function is repeated so I suggest to extract it as a named one and reuse it.

**Review by faustocv — CHANGES_REQUESTED** [review]
Looks good but it has room for improvement

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts (hunk 2 above)
Comment (edgardavid2015) [reply src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts]: Thanks for your review @faustocv. This change was applied in the last commit.

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 3]
```diff
@@ -83,4 +94,10 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
     const regex = new RegExp('http*', 'g');
     return regex.test(uri);
   }
+
+  bodyExportOptions(data, row, column, node){
+    console.log(data);
```
Comment (faustocv) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:99]: Please remove this line

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.spec.ts [hunk 4]
```diff
@@ -107,6 +112,14 @@ describe('Reports Page', () => {
       });
     });
 
+    it('The data should be displayed as a multiple of hour when column is equal to 3', () => {
+      expect(component.bodyExportOptions(durationTime, 0, 3, 0)).toMatch(/^\d+\.\d{0,2}$/);
```
Comment (faustocv) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.spec.ts:116]: Could you assign this values: 0, 3, 0 and the regex expression with variables, please? It's difficult to understand what they are about

**Review by faustocv — CHANGES_REQUESTED** [review]

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.spec.ts (hunk 4 above)
Comment (edgardavid2015) [reply src/app/modules/reports/components/time-entries-table/time-entries-table.component.spec.ts]: These changes will be in the next commit, thank you @faustocv.

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts (hunk 3 above)
Comment (edgardavid2015) [reply src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts]: sorry for that. This line is deleted in the new commit.

**Review by JosueOb — APPROVED** [review]

**Review by faustocv — APPROVED** [review]
Now it's looking better.

**Pull request merged**